### ISSUE TITLE
SQL CE + AmbientTransactionWarning

### ISSUE THREAD

**Issue by optiks** [ErikEJ/EntityFramework.SqlServerCompact#544]
Hi,

Using EF Core with this provider and an ambient transaction generates the warning:
```c#
Microsoft.EntityFrameworkCore.Database.Transaction.AmbientTransactionWarning: An ambient transaction has been detected. The current provider does not support ambient transactions. See http://go.microsoft.com/fwlink/?LinkId=800142'. This exception can be suppressed or logged by passing event ID 'RelationalEventId.AmbientTransactionWarning' to the 'ConfigureWarnings' method in 'DbContext.OnConfiguring' or 'AddDbContext'
```

Looking through the EF Core source code, it looks like [SupportsAmbientTransactions](https://github.com/aspnet/EntityFrameworkCore/blob/master/src/EFCore.Relational/Storage/RelationalConnection.cs#L153) may need to be set to `true`.

Our workaround for the moment is to suppress this message, however it would be good to remove that workaround if possible.

**Comment by ErikEJ** [comment]
Limitations of SQL CE and TransactionScope

https://technet.microsoft.com/en-us/library/bb896149(v=sql.110).aspx

**Comment by optiks** [comment]
Hi Erik, just to confirm, is the nightly [2.1.0-rc1-12](https://www.myget.org/feed/ef7-sqlce/package/nuget/EntityFrameworkCore.SqlServerCompact35/2.1.0-rc1-12)? The [https://www.myget.org/feed/ef7-sqlce/package/nuget/EntityFrameworkCore.SqlServerCompact35](MyGet feed) is showing 2.1.0-rc1-9 as the most recent.

**Comment by ErikEJ** [comment]
Yeah, the sorting in the MyGet UI is not perfect - rc1-12 is the latest (and based on 2.1.0 RTM)

**Comment by ErikEJ** [comment]
I have just released 2.1.0.2 on NuGet.org

**Comment by optiks** [comment]
Many thanks. We’ve got a story to try this out, but haven’t got to it yet sorry. We’ll do it next week.

**Comment by optiks** [comment]
Sorry, I dropped the ball on this one. I'll try and get to it over the next couple of days.
